core/domain/param_domain.py

```diff
@@ -57,25 +86,58 @@ class ParamChange(object):
     """Value object for a parameter change."""
 
     def __init__(self, name, generator_id, customization_args):
-
+        """Initialize a ParamChange object with the specified arguments.
+
+        Args:
+            name: unicode. The name of the parameter.
+                generator_id: unicode. The type of generator used to create the
```

Comment:
The indentation of 'name', 'generator_id' and 'customization_args' (the args of the method) should all be at the same level.